.github/workflows/update_changelog.yml

```
@@ -0,0 +1,40 @@
+    tags:
+      - 'v*.*.*'
+    branches:
+      - chore/github-workflow-changelog-update
+
+jobs:
+  update-changelog:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)